chipsec/utilcmd/test_id_cmd.py

```diff
@@ -0,0 +1,74 @@
+# CHIPSEC: Platform Security Assessment Framework
+# Copyright (c) 2024, Intel Corporation
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; Version 2.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# Contact information:
+# chipsec@intel.com
+#
+
+"""
+Generate a test ID using hashlib from the test's file name (no file extension).
+Hash is truncated to 28 bits. For test names use full path as seen in example below.
+
+Usage:
+    ``chipsec_util  name <test name>``
+    ``chipsec_util hash <test hash>``
+
+Examples:
+    >>> chipsec_main.py -m common.tools.generate_test_id name chipsec.modules.common.remap
+    >>> chipsec_main.py -m common.tools.generate_test_id hash 0x67eb58d
+"""
+
+from chipsec.library.returncode import get_module_ids_dictionary, generate_hash_id
+from chipsec.command import BaseCommand, toLoad
+from argparse import ArgumentParser
+
+
+class TestIdCommand(BaseCommand):
```

Comment:
Should probably be "ModuleIdCommand", when I saw "test", I thought this was a unit test. (Same with the filename)